doc/api/buffer.md

```diff
@@ -1778,11 +1853,12 @@ for (var value of buf) {
 * `encoding` {String} Default: `'utf8'`
 * Return: {Number} Numbers of bytes written
 
-Writes `string` to the Buffer at `offset` using the given `encoding`.
-The `length` parameter is the number of bytes to write. If the Buffer did not
-contain enough space to fit the entire string, only a partial amount of the
-string will be written however, it will not write only partially encoded
-characters.
+Writes `string` to `buf` at `offset` according to the character encoding in `encoding`.
+The `length` parameter is the number of bytes to write. If `buf` did not contain
+enough space to fit the entire string, only a partial amount of the string will
+be written however, it will not write only partially encoded characters.
```

Comment:
not a native speaker but this sounds like there should be a semicolon or full stop before `however`?